ansible/roles_ocp_workloads/ocp4_workload_rhte19_shadowman/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+become_override: false
+ocp_username: opentlc-mgr
+
+ocp_project: demojam
```

Comment:
Variables need to be prefixed with the role name:
e.g. ocp4_workload_rhte19_shadowman_ocp_project: demojam